k8s.gcr.io/images/k8s-staging-sig-storage/images.yaml

```diff
@@ -101,7 +101,7 @@
     "sha256:6029c252dae6178c99b580de72d7776158edbc81be0de15cedc4152a3acfed18": [ "v1.7.3" ]
 - name: iscsiplugin
   dmap:
-    "sha256:31117074c183242b223340f2fc0f886243e97c6cb1a83d98cfb00a69d1746697": [ "v0.1.0" ]
+    "sha256:31117074c183242b223340f2fc0f886243e97c6cb1a83d98cfb00a69d1746697": [ "0.1-canary;v0.1.0" ]
```

Comment:
Did this come out of the `generate.sh` script? Are you sure that this syntax is valid?

We usually don't promote canary images, so isn't just `v0.1.0` enough?

Reply:
@pohly it came out of `generate.sh` ( may be a confirmation from your end helps too)  .. so I believe this syntax is valid too..  due to a mistake in the hash recorded here for iscsiplugin , we ( with the help of @msau42 ) adjusted the tag in the backend repo  , thats why both of these in place..

Reply:
https://github.com/kubernetes/k8s.io/blob/289fd8c5189e3cbff76ce0d5bf4fc035692da306/k8s.gcr.io/images/k8s-staging-sig-storage/generate.sh#L56

just passes through what it gets from gcloud, which seems to be `0.1-canary;v0.1.0` in this case now. I have no idea whether this is valid in `images.yaml`.

Reply:
hmmmm I'm not sure if the promotion tool will work with this. Also we should not promote canary tags. Probably we need to fix generate.sh